modules/apps/frontend-theme/frontend-theme-cms/src/css/components/_sticker.scss

```diff
@@ -0,0 +1,15 @@
+.cms-sticker-dropdown {
```

Comment:
@pat270 would this make sense to be added via the `sticker-palette`?

Reply:
@marcoscv-work could you share the ticket for adding interactive styles to the sticker component?